src/lib/models/useDeleteModelMutation.ts

```diff
@@ -0,0 +1,20 @@
+import { useDataEngine } from '@dhis2/app-runtime'
+import { useMutation } from 'react-query'
+import { Schema } from '../useLoadApp'
+
+type MutationFnArgs = {
+    schema: Schema
+    id: string
+}
+
+export function useDeleteModelMutation() {
+    const engine = useDataEngine()
+    const mutationFn = async (variables: MutationFnArgs) =>
+        engine.mutate({
```

Comment:
Should handle the error. Probably use an alert and show it. Also think we should have an alert for success.